catchup/universalFetcher.go

```diff
@@ -195,7 +198,11 @@ func (w *wsFetcherClient) requestBlock(ctx context.Context, round basics.Round)
 // set max fetcher size to 10MB, this is enough to fit the block and certificate
 const fetcherMaxBlockBytes = 10 << 20
 
-var errNoBlockForRound = errors.New("No block available for given round")
+type noBlockForRoundError struct {
+	latest, round basics.Round
+}
+
+func (noBlockForRoundError) Error() string { return "No block available for given round" }
```

Comment:
```suggestion
func (noBlockForRoundError) Error() string { return "no block available for given round" }
```
I think the linter might complain about capitalized err messages?